web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -81,6 +81,10 @@
           'isLogoInHeader': false,
           'logoInHeaderPosition': 'left'
         },
+        'footer': {
```

Comment:
This settings implies that one can also disable the footer entirely. Not sure this is the intent?

Reply:
No, that's not what I intended to do. I just wanted to make a new section for the footer.

Reply:
I will have a look for a better location

Reply:
It would make sense to be able to hide or not the footer no ? Some user embed GeoNetwork in other website which contains header/footer materials.

Reply:
Yes, it could be useful, but it was not really what I intended in this PR. I was just looking for a good place in the UI to add the setting.

Reply:
Maybe organize it with https://github.com/geonetwork/core-geonetwork/pull/3529 which add the footer to all apps ?

Reply:
You mean combine the 2 PR's?

Reply:
We could merge [3529](https://github.com/geonetwork/core-geonetwork/pull/3529) and then apply this one which allows to turn on/off the footer ?